builtin/credential/aws/pkcs7/sign_test.go

```diff
@@ -151,6 +151,7 @@ func TestDSASignAndVerifyWithOpenSSL(t *testing.T) {
 		"-content", tmpContentFile.Name())
 	out, err := opensslCMD.CombinedOutput()
 	if err != nil {
+		t.Errorf("Command: openssl smime -verify -noverify -in %s -inform PEM -content %s", tmpSignatureFile.Name(), tmpContentFile.Name())
```

Comment:
Can this use opensslCMD.Args instead of duplicating the command? Something like:

```suggestion
		t.Errorf("Command: %s", opensslCMD.Args)
```

Reply:
ah, that works. (I had this to help with debugging.)